Resources/public/css/styles.css

```diff
@@ -496,3 +496,15 @@ div.sonata-filters-box div.form-group span.input-group-addon {
     padding: 3px 10px;
     font-size: 13px;
 }
+
+.hide {
+    display: none;
+}
+
+.fade {
```

Comment:
@muspelheim after you applied this comment, this feature looks ready to merge to me